src/styles/header.scss

```
@@ -0,0 +1,16 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    position: relative;
+    width: 40px;
+    height: 40px;
+    left: 50px;
```

```suggestion
    position: relative;
    left: 50px;
    width: 40px;
    height: 40px;
```
Remember about [styles properties order](https://codeguide.academy/html-css.html#css-order)